source/flux/parts/stoprint.js

```diff
@@ -85,9 +89,18 @@ export function updatePrinters(): ThunkAction<UpdateAllPrintersAction> {
 
 		const {recentPrinters, popularPrinters} = recentAndPopularPrinters
 
+		const colorPrintersUrl = API('/printing/color-printers')
+		const {data} = await fetchJson(colorPrintersUrl).catch(err => {
```

Comment:
You should move this fetch into the Promise.all, do it can be executed simultaneously with the other two network requests, instead of sequentially afterwards.